tests/lib/ecma-features.js

```diff
@@ -41,8 +41,8 @@ var assert = require("chai").assert,
 // Setup
 //------------------------------------------------------------------------------
 
-var FIXTURES_DIR = "./tests/fixtures/ecma-features",
-    FIXTURES_MIX_DIR = "./tests/fixtures/ecma-features-mix";
+var FIXTURES_DIR = "./tests/fixtures/ecma-features";
+// var FIXTURES_MIX_DIR = "./tests/fixtures/ecma-features-mix";
```

Comment:
I didn't come in with any expectations, given the alpha version, but to be honest... nothing about this project has worked so far :smile: 

Specifically for the tests not running, the first issue is as described in https://github.com/eslint/typescript-eslint-parser/issues/6

Each PR I have submitted so far has been about trying to pick off the next lowest-hanging fruit, and, given it doesn't really require any domain-specific knowledge, fixing the linting seemed like the best choice at this stage. The tests will be my next focus.